tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs

```diff
@@ -43,21 +47,641 @@ public class SanitizerDictionary
             ResetSessionSanitizers();
         }
 
+        /*
+         * The below list has been grouped and labelled with some room for expansion. As such:
+         * 
+         * General sanitizers  = 1XXX
+         * Header sanitizers   = 2XXX
+         * Body sanitizers     = 3XXX
+         * URI, special, other = 4XXX
+         * 
+         * */
+
+        private const string EMPTYGUID = "00000000-0000-0000-0000-000000000000";
+
         public List<RegisteredSanitizer> DefaultSanitizerList = new List<RegisteredSanitizer>
             {
+                #region GeneralRegex
+                // basic RecordedTestSanitizer handles Authorization header
                 new RegisteredSanitizer(
                     new RecordedTestSanitizer(),
-                    "AZSDK001"
+                    "AZSDK0000"
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "SharedAccessKey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    "AZSDK1000"
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "AccountKey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    "AZSDK1001"
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "accesskey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    "AZSDK1002"
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "Accesskey=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    "AZSDK1003"
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "Secret=(?<key>[^;\\\"]+)", groupForReplace: "key"),
+                    "AZSDK1004"
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "common/userrealm/(?<realm>[^/\\.]+)", groupForReplace: "realm"),
+                    "AZSDK1005",
+                    "ACS Identity leverages these strings to store identity information."
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "/identities/(?<realm>[^/?]+)", groupForReplace: "realm"),
+                    "AZSDK1006",
+                    "ACS Identity leverages these strings to store identity information."
+                ),
+                new RegisteredSanitizer(
+                    new GeneralRegexSanitizer(regex: "(?:[?&](sv|sig|se|srt|ss|sp|st)=)(?<secret>[^&\\\"\\s]*)", groupForReplace: "secret"),
```

Comment:
Suggest removing "se" as this is an expiration time.

Reply:
Actually.. there are a bunch of other parameters that break .NET tests - essentially everything other than "sig" and "sv". I don't think the other parameters are secrets or even PII.

Reply:
Agreed. @pvaneck actually suggested we add one, but going the opposite direction will also help him.

So remove everything other than `sv|sig` from this?

Reply:
Eh posted a discussion item.

Reply:
After offline discussion reducing to just `sig`.